src/dropdown/Dropdown.tsx

```diff
@@ -149,11 +149,7 @@ function Dropdown<OptionIdShape extends string>({
         {selectedOption ? selectedOption.title : deselectOptionTitle || placeholder}
       </span>
 
-      {areOptionsFetching ? (
-        <Spinner spinnerColor={"black"} backgroundColor={"#EBEBEB"} />
-      ) : (
-        <CaretDownIcon aria-hidden={true} />
-      )}
+      {areOptionsFetching ? <Spinner /> : <CaretDownIcon aria-hidden={true} />}
```

Comment:
Let's pass "dropdown__spinner" as a customClassName to Spinner. Also can we allow passing a customSpinner to Dropdown as well?

Reply:
Why not? I think we can provide this with a `customSpinner` prop.